scripts/cli-test-env.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
+
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ "$SOURCE" != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
+done
+
+REPO_ROOT="$( cd -P "$( dirname "$SOURCE" )/../" && pwd )"
+
+RID=linux-x64
```

Comment:
How does this work on Mac?

Reply:
bash shell behavior is different between mac and Linux? which specific part?

Reply:
Shouldn't the "RID" be determined programmatically?

Reply:
Something like this maybe:

RID=linux-x64 
if [ "$uname" = "Darwin" ]; then
    RID=osx-x64